packages/dev-tools-pages/ts/components/Typography.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+
+import { media } from '../variables';
+
+const Alpha = styled.h2`
+    font-size: 1.75rem;
+    line-height: 1;
+
+    ${media.small`font-size: 1.5rem;`};
+`;
+
+const Beta = styled.h3`
+    font-size: 1.25rem;
+    line-height: 1.65;
+`;
+
+const Gamma = styled.h4`
+    font-size: 1rem;
+
+    ${media.small`font-size: 0.875rem;`};
+`;
+
+const Lead = styled.p`
+    font-size: 1.25rem;
+    line-height: 1.6;
+    ${media.small`font-size: 1rem;`};
+`;
+
+const Small = styled.p`
+    font-size: 0.875rem;
+`;
+
```

Comment:
This file is cool is there an article on what you're doing here?

Reply:
No, not really. It's just the way I would have done it with css classes, setting up common font sizes in one file :)